model_compression_toolkit/hardware_models/default_hwm.py

```diff
@@ -0,0 +1,107 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import model_compression_toolkit as mct
+hwm = mct.hardware_representation
+
+
+def get_default_hardware_model():
+    # Create a quantization config.
+    # A quantization configuration defines how an operator
+    # should be quantized on the modeled hardware:
+    eight_bits = hwm.OpQuantizationConfig(
+        activation_quantization_method=hwm.QuantizationMethod.POWER_OF_TWO,
+        weights_quantization_method=hwm.QuantizationMethod.POWER_OF_TWO,
+        activation_n_bits=8,
+        weights_n_bits=8,
+        weights_per_channel_threshold=True,
+        enable_weights_quantization=True,
+        enable_activation_quantization=True
+    )
+
+    # Create a QuantizationConfigOptions, which defines a set
+    # of possible configurations to consider when quantizing a set of operations (in mixed-precision, for example).
+    # If the QuantizationConfigOptions contains only one configuration,
+    # this configuration will be used for the operation quantization:
+    default_configuration_options = hwm.QuantizationConfigOptions([eight_bits])
+
+    # Create a HardwareModel and set its default quantization config.
+    # This default configuration will be used for all operations
+    # unless specified otherwise (see OperatorsSet, for example):
+    default_hwm = hwm.HardwareModel(default_configuration_options, name='default_hwm')
+
+    # To start defining the model's components (such as operator sets, and fusing patterns),
+    # use 'with' the hardware model instance, and create them as below:
+    with default_hwm:
+        # Create an OperatorsSet to represent a set of operations.
+        # Each OperatorsSet has a unique label.
+        # If a quantization configuration options is passed, these options will
+        # be used for operations that will be attached to this set's label.
+        # Otherwise, it will be a configure-less set (used in fusing):
+
+        # May suit for operations like: Dropout, Reshape, etc.
+        hwm.OperatorsSet("NoQuantization",
+                         hwm.get_default_quantization_config_options().clone_and_edit(
+                             enable_weights_quantization=False,
+                             enable_activation_quantization=False))
+
+        # To quantize a model using mixed-precision, create
+        # a QuantizationConfigOptions with more than one QuantizationConfig.
+        # In this example, we quantize some operations' weights
+        # using 2, 4 or 8 bits, and when using 2 or 4 bits, it's possible
+        # to quantize the operations' activations using LUT.
+        four_bits = eight_bits.clone_and_edit(weights_n_bits=4)
+        two_bits = eight_bits.clone_and_edit(weights_n_bits=2)
+        four_bits_lut = four_bits.clone_and_edit(
```

Comment:
Remove the LUT only weights quantization and 8,4,2 MP

Reply:
Fixed